entity-service-api/src/main/proto/org/hypertrace/entity/query/service/v1/entity_query_request.proto

```diff
@@ -50,3 +50,18 @@ message SetAttribute {
   ColumnIdentifier attribute = 1;
   LiteralConstant value = 2;
 }
+
+message BulkEntityArrayAttributeUpdateRequest {
+  string entityType = 1;
+  repeated string entityIds = 2;
+  ColumnIdentifier attribute = 3;
```

Comment:
it may make sense to pull the operation into its own message and take multiple so we can make multiple updates at once. Would the lower level api support that? If not, I guess we can defer that and roll it into my dream of a more complex bulk update api.

Reply:
Not in its current form.

Also the frontend APIs may be scoped smaller and one at the backend(data store one) may handle multiple of these smaller bulk operations